bin/cmd.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+import WikibaseRepo from './../lib/repo.js'
+import { pickLanguages, pickKeys } from './../lib/util.js'
+import { execCmd } from './../lib/exec.js'
+
+;(async () => {
+  const [source, target, ...entities] = process.argv.slice(2)
+  const sourceRepo = new WikibaseRepo(source)
+  const targetRepo = new WikibaseRepo(target, {
+    oauth: {
+      consumer_key: process.env.TARGET_WIKI_OAUTH_CONSUMER_TOKEN,
+      consumer_secret: process.env.TARGET_WIKI_OAUTH_CONSUMER_SECRET,
+      token: process.env.TARGET_WIKI_OAUTH_ACCESS_TOKEN,
+      token_secret: process.env.TARGET_WIKI_OAUTH_ACCESS_SECRET
+    }
+  })
+
+  const contentLanguages = await targetRepo.getContentLanguages()
+
+  let data = await sourceRepo.getEntities(...entities)
+  data = data
+    .map(e => pickKeys(e, 'type', 'labels', 'descriptions', 'aliases', 'datatype'))
+    .map(e => pickLanguages(e, ...contentLanguages))
+
+  await targetRepo.createEntities(...data)
+  return `Sucessfully transferred ${entities.length} entities from ${source} to ${target}.`
+})()
+  .then((result) => {
+    if (result) {
+      console.log(result)
+    }
+  })
+  .catch((err) => {
+    console.error(err)
+    process.exitCode = 1
+  })
+  .then(async () => {
+    if (process.env.CALLBACK_ON_SUCCESS && process.exitCode === 0) {
+      await execCmd(process.env.CALLBACK_ON_SUCCESS)
+    }
+    if (process.env.CALLBACK_ON_FAILURE && process.exitCode !== 0) {
+      await execCmd(process.env.CALLBACK_ON_FAILURE)
+    }
+  })
+  .catch((err) => {
```

Comment:
Interesting question: if the transfer itself worked, but the callback failed, the script will now still exit 0. Is this what we want? I'm unsure.

Reply:
After trying to think through the steps that would happen in this case I believe it's probably no worse than returning `1`.

In either case because of how we specified the spec the job won't retry.

This means that it may be marginally less clear to the debugging operator but on the other hand the "main work" done by the job did succeed.